src/OMSimulatorLib/Model.cpp

```diff
@@ -478,6 +572,10 @@ oms_status_enu_t oms::Model::exportToFile(const std::string& filename) const
   //        -9  Compress better
   //        -j  exclude path. store only the file name
   std::vector<std::string> resources;
+  // TODO, export ssv file to resource directory
+  if (parameterNodeCount > 0)
```

Comment:
Shouldnb't you check for the filename? E.g. `if (!ssvFileName.empty())`

Reply:
Yes we can check also for ssvFileName is empty,  but both are doing the same, i change to check for ssvFile